.gitignore

```diff
@@ -0,0 +1,2 @@
+/nbproject/private/

```

Comment:
GitIgnore is a file which states which files shouldn't be uploaded when you do a push to GitHub. The review function is for when you are working in groups or looking at someone elses code and you'd like to make a comment about a specific file from their code.